src/displayBackend/wayland/Seat.hpp

```diff
@@ -28,6 +28,9 @@ namespace Wayland {
 class Seat : public Registry
 {
 public:
+	//  Wayland 1.11
+	//	static const int cVersion = 5;
```

Comment:
can we detect wayland version?

Reply:
Handling registry version is still open question need to be investigated.